xml/obs_glossary.xml

```diff
@@ -72,30 +72,32 @@
  </glossentry>
  <!-- ############## B ############## -->
  <glossentry xml:id="obs.glos.project_binary">
-  <glossterm>Binary Package</glossterm>
+  <glossterm>Binary Package (Binary)</glossterm>
   <glossdef>
    <para>
-    A binary package (in the &obs; context often called a binary) is
-    an archive file that contains an installable version of software and
+    An archive file that contains an installable version of software and
     metadata. Binary packages can also include, for example, configuration
     files and documentation.
    </para>
    <para>
-    Binary packages are one possible result of building on &obsa;.
-    &obsa; supports building multiple types of binary package. Examples
-    include &rpmf; packages and &debf; packages.
+    Unlike containers, binary packages do not include dependencies of the
+    main software. Instead, dependencies are packaged as additional binary
+    packages. Formats of binary packages include &rpmf; and &debf;.
```

Comment:
I think this sentence could be a bit misleading. Actually, there are some *meta data/pointer/references* to dependencies, but not the referenced dependencies itself.

To make this distinction more prominent, I would suggest something like this:

```
Unlike containers, binary packages do only contain meta data about the dependencies, but do not
include them [directly]. Instead, dependencies are packaged as additional binary packages. [...]
```